config/deploy/production.rb

```diff
@@ -17,7 +17,7 @@
 # server list. The second argument is a, or duck-types, Hash and is
 # used to set extended properties on the server.
 
-server 've.eyequeue.us', user: 'deploy', roles: %w{web app db}
+server 've.eyequeue.us', user: 'deploy', roles: %w{web app}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.